build.sbt

```
@@ -57,7 +57,8 @@ scalacOptions ++= Seq(
   "-language:reflectiveCalls",
   "-deprecation",
   "-feature",
-  "-Xcheckinit"
+  "-Xcheckinit",
+  "-Wconf:msg=firrtl:s", // do not warn about firrtl imports
```

```suggestion
  "-Wconf:msg=Importing from firrtl:s", // do not warn about firrtl imports
```
I now realize this is what we're using in Chisel itself, but IMO this message is a little too broad. I would suggest a slightly more specific message, but it's up to you!

---

I would prefer to stick to Adam's example.

---

I think you were actually right ... sorry. I am going to add this to the other warnings PR. Now that I understand better how this works, seems like it would be better to be more specific.